teps/0021-results-api.md

```diff
@@ -457,11 +457,6 @@ message Result {
   // Arbitrary user provided labels for the result.
   map<string, string> annotations = 3;
 
-  // Execution artifacts part of this result. Typically will be Tekton
-  // Task/PipelineRuns, but may also include other execution information
-  // (e.g. DSLs, etc.)
-  repeated Event events = 4;
-
   // The etag for this result.
   // If this is provided on update, it must match the server's etag.
   string etag = 6
```

Comment:
NIT: should this become #4 now?

Reply:
Technically for protos you should never reuse field numbers and just mark them as `reserved`. For proposal docs, this doesn't matter too much - I'll be reindexing these once this proto becomes real.